src/app/zap-templates/templates/chip/CHIPClusters-src.zapt

```diff
@@ -2,7 +2,7 @@
 #include "CHIPClusters.h"
 
 #include <app/chip-zcl-zpro-codec-api.h>
-#include <app/im-encoder.h>
+#include <app/im-handler.h>
```

Comment:
The encoding code does not really depends on the handling side. I guess you want to use the appropriate headers.